nservicebus/messaging/publish-subscribe/index.md

```diff
@@ -34,6 +34,72 @@ When requested by applicative logic to publish a message, the NServiceBus infras
 
 Since one-way messaging is used to dispatch physical messages, even if one of the subscriber endpoints is offline or otherwise unavailable, this does not cause the publishing thread to block. The message is stored in the sending machine's outgoing queue (for a configurable period of time), while the messaging infrastructure attempts to deliver the message to its destination.
 
+## What is auto subscribe
+
+The default mode auto subscribe. When a subscriber endpoint is started it determines to which events it needs to subscribe. It then sends sends subscribtion messages to the dependent publishers. The publishers update their subscription data.
```

Comment:
> The publishers update their subscription data.

"subscription data" seems to imply "who the publishers subscribe to" when actually you mean the inverse